src/Main.java

```diff
@@ -1,5 +1,235 @@
+        // da popunite niz sa int brojevima.
+        // Zatim ispisite sve brojeve po jedan broj po redu (println) ,
+        // na ispisu svih brojeva kod mora ispisati obrnuti  redosljed(ne index) u nizu ,
+        // te da li je broj paran ili ne kao i da li je veci od 10.
+        Scanner reader = new Scanner(System.in);
+        // int[] myIntArray = {1, 2, 3, 4};
+        System.out.println("Koliki niz zelite? ");
+
+        // Console READ user value
+        int lengthOfArray = reader.nextInt();
+        int[] numberOfIntegers = new int[lengthOfArray];
+
+        // enter and populate elements
+        for (int i = 0; i < lengthOfArray; i++) {
+            System.out.println("Unesite element niza ?");
+            int elementOfArray = reader.nextInt();
+            numberOfIntegers[i] = elementOfArray;
+        }
+        // print elements
+        for (int i = lengthOfArray; i > 0; i--) {
+            String description = "";
+            if (numberOfIntegers[i - 1] % 2 == 0) {
+                description = " - paran broj";
+            } else {
+                description = " - neparan broj";
+            }
+            if (numberOfIntegers[i - 1] > 10) {
+                description += " i veci od 10";
+            } else {
+                description += " i manji od 10";
+            }
+            System.out.println(numberOfIntegers[i - 1] + description);
+        }
+
+        // Task 2 Napravite program koji moze za niz od 10 brojeva da ispise:
+        //First number in array: 123
+        //Second number in array: 23
+        //Third number in array: 3
+        //itd itd.
+        //i da se ovakav ispis moze koristiti do 15 elementa u nizu.
+
+        int[] arrayOfNums = {123, 23, 10, 240, 117, 33, 78, 65, 32, 44, 91, 24, 93, 4, 5};
+        String[] positionInArray = {"First", "Second", "Third", "Fourth", "Fifth", "Sixth", "Seventh",
+                "Eighth", "Ninth", "Tenth", "Eleventh", "Twelfth", "Thirteenth", "Fourteenth", "Fifteenth"};
+
+        for (int i = 0; i < arrayOfNums.length; i++) {
+            System.out.printf("%s number in array: %s", positionInArray[i], arrayOfNums[i]);
+            System.out.println('\n');
+        }
+
+        //task 3 Napravite program koji ce obrnuti redosljed niza odnosno:
+        //1 2 3 4 5 6 7   postaje 7 6 5 4 3 2 1
+        //i da moze primiti bilo koji niz i obrnuti ga.
+
+        System.out.println("Koji niz zelite? ");
+        //Console READ user value
+        int lengthOfArray1 = reader.nextInt();
+        int[] numberOfIntegers1 = new int[lengthOfArray1];
+
+        // enter and populate elements
+        for (int i = 0; i < lengthOfArray1; i++) {
+            System.out.println("Unesite elemente niza ?");
+            int elementOfArray1 = reader.nextInt();
+            numberOfIntegers1[i] = elementOfArray1;
+        }
+        // print elements
+        for (int i = lengthOfArray1; i > 0; i--) {
+            System.out.println(numberOfIntegers1[i - 1]);
```

Comment:
it was not a task to print out the array in reverse but to reverse the array itself